dist/index.html

```diff
@@ -1,57 +1,17 @@
 <!DOCTYPE html>
 <html lang="en" class="dark-theme">
   <head>
-    <title>Abundance</title>
+    <title>Abundance - BETA</title>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="icon"
-      href="/abundance-mo-fork/assets/abundance_logo-BkAK_rm1.png"
-    />
+    <link rel="icon" href="/assets/abundance_logo-BkAK_rm1.png" />
     <!-- should change for deployment probably -->
     <!-- <link rel="stylesheet" href="https://unpkg.com/mvp.css"> -->
-    <!-- Start Single Page Apps for GitHub Pages -->
```

Comment:
@alzatin Do you think that these changes make sense?

Reply:
Yeah, I think so. I don't think we use those redirects anymoe . Wait so is everything AI or you made some changes to the AI PR?